sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -205,6 +210,7 @@ public AutoConfiguredOpenTelemetrySdk build() {
 
     if (setResultAsGlobal) {
       GlobalOpenTelemetry.set(openTelemetrySdk);
+      logger.log(Level.FINE, "Global OpenTelemetrySdk set to {0}", openTelemetrySdk);
```

Comment:
Should this line go in `GlobalOpenTelemetry` instead?

Reply:
Hmm - not sure. I guess the logging should generally not be that helpful if a user is constructing their own SDK, it's really for resolving AutoConfiguration past all SPIs etc. Lean towards keeping it here.

Reply:
Yeah I buy that. Can we add some sort of indication that autoconfigure is the thing that is setting it? Something like "AutoConfigure setting Global OpenTelemetrySdk to {0}".

Also might be helpful to do this at a higher logging level. Pretty useful to know how your environment variables / system properties translated to the sdk config.

Reply:
I added autoconfiguration to the log message. I've left the level at least for now, if nothing else the output looks like sort of goop without any newlines / indenting so it might be a bit scary to log by default but we can revisit later I think.